datafusion/sql/src/unparser/plan.rs

```diff
@@ -347,8 +373,33 @@ impl Unparser<'_> {
 
                 Ok(())
             }
-            LogicalPlan::Union(_union) => {
-                not_impl_err!("Unsupported operator: {plan:?}")
+            LogicalPlan::Union(union) => {
```

Comment:
I double checked that Union is UNION ALL https://docs.rs/datafusion/latest/datafusion/logical_expr/struct.Union.html